lib/topic.ts

```diff
@@ -0,0 +1,35 @@
+import { expect, Page } from '@playwright/test';
+import { waitForKafkaReady } from './kafka';
+import { config } from './config';
+
+export const navigateToKafkaTopicsList = async function (page: Page, kafkaName: string) {
+  await expect(page.getByText(kafkaName)).toHaveCount(1);
+  await waitForKafkaReady(page, kafkaName);
+  await page.getByText(kafkaName).click();
+  await page.getByText('Topics').click();
+};
+
+export const createKafkaTopic = async function (page: Page, name: string) {
+  await page.getByText('Create topic').click();
+  await expect(page.getByText('Create topic')).toHaveCount(2);
+  // This is default Topic creation
+  await page.getByPlaceholder('Enter topic name').fill(name);
+  await page.getByText('Next').click();
+  await page.getByText('Next').click();
+  await page.getByText('Next').click();
+  await page.getByText('Finish').click();
+
```

Comment:
can you add an expectation here that the topic has been created?

```ts
await expect(page.getByText(name)).toHaveCount(1);
```

Reply:
added